closed/test/jdk/openj9/internal/security/TestProperties.java

```diff
@@ -164,6 +166,28 @@ private static Stream<Arguments> patternMatches_expectedExitValue1() {
         );
     }
 
+    private static Stream<Arguments> patternMatches_PropertiesList() {
+        return Stream.of(
+                Arguments.of("Test-Profile-Property-List.A",
+                        System.getProperty("test.src") + "/propertyListA-java.security",
+                        "(?s)(?=.*OpenJCEPlusFIPS)(?=.*Sun)(?=.*SunJSSE)(?=.*SunEC)",
```

Comment:
Some comments would be welcome explaining patterns such as this - it's not clear (to me anyway) what this is intended to match.

Reply:
I suggest the comments need not repeat `"-Djava.security.propertiesList=" test` and should follow the style of existing comments. So, for example, this would become:
```java
                // 1 - The profile in propertyListA-java.security extends the profile
                // in the main java.security file, which lists 4 providers.
```
Notice also the comment starts with a capital and ends with a period (again like pre-existing comments).